packages/@ama-sdk/core/src/plugins/bot-protection-fingerprint/bot-protection-fingerprint.spec.ts

```diff
@@ -270,7 +270,7 @@ describe('BotProtectionFingerprint', () => {
       expect(fingerprintRetriever).toHaveBeenCalledTimes(1);
     });
 
-    it('Should\' add the fingerprint header if no fingerprint after the configured poller is done running.', async () => {
+    it('Should add the fingerprint header if no fingerprint after the configured poller is done running.', async () => {
```

Comment:
should add the fingerprint header if theire is no fingerprint once the configured poller has finished (done eventually) running.